src/models/db/transfers.js

```diff
@@ -90,6 +90,20 @@ function * getTransfer (uuid, options) {
     })
 }
 
+function * getTransferByExecutionCondition (executionCondition, options) {
```

Comment:
I actually already did this in my pull request, so you could copy from there: https://github.com/interledger/five-bells-ledger/pull/286/files#diff-fb6e03a84a05cfd77df4bcbfa0e28b33R92